src/ixdat/plotters/ec_plotter.py

```diff
@@ -37,18 +42,19 @@ def plot_measurement(
             measurement (Measurement): The measurement to plot, if not the one the
                 plotter was initiated with.
             tspan (iter of float): The timespan (wrt to measurement.tstamp) to plot.
-            v_name (string): The name of the ValueSeries to plot on the left y-axis.
+            axes (list of matplotlib.Axis): Two axes to plot on, if not the default
+                new twinx()'d axes. axes[0] is for `U_name` and axes[1] for `J_name`.
+            U_name (string): The name of the ValueSeries to plot on the left y-axis.
                 Defaults to measurement.V_str, which for an ECMeasurement is the name
                 of its most calibrated/correct potential.
-            j_name (string): The name of the ValueSeries to plot on the right y-axis.
+            J_name (string): The name of the ValueSeries to plot on the right y-axis.
                 Defaults to measurement.J_str, which for an ECMeasurement is the name
                 of its most normalized/correct current.
-            axes (list of matplotlib.Axis): Two axes to plot on, if not the default
-                new twinx()'d axes. axes[0] is for V_str and axes[1] for J_str.
-            V_str (str): DEPRECIATED. now v_name
-            J_str (str): DEPRECIATED. now j_name
-            v_color (str): The color to plot v_name. Defaults to black.
-            j_color (str): The color to plot j_name. Defaults to red.
+            U_color (str): The color to plot U_name. Defaults to black.
+            J_color (str): The color to plot J_name. Defaults to red.
+            V_str (str): DEPRECATED. Use `U_name`.
+            J_str (str): DEPRECATED. Use `J_name`.
+            V_color (str): DEPRECATED. Use `U_color`.
```

Comment:
Is that "Use `v_color`"? At least that's what's in the decorator

Reply:
`v_color` was a rejected intermediate between `V_color` and `U_color` which never saw PyPI. (Same with `V_name`.) Decorator fixed.